app.js

```diff
@@ -1,9 +1,15 @@
 const express = require('express');
 const app = express();
 const port = process.env.PORT || 3000;
+const multer = require('multer')
+const upload = multer({
+  dest: 'static/img/'
+})
 
 app.set('view engine', 'ejs');
 app.use(express.static('static'));
+app.get('/add', profileform)
```

Comment:
Tiny edit: Let's use `camelCase` for names with multiple words.

```suggestion
app.get('/add', profileForm)
```